src/widget/tool/screenshotgrabber.cpp

```diff
@@ -76,7 +76,14 @@ void ScreenshotGrabber::showGrabber()
     this->window->show();
     this->window->setFocus();
     this->window->grabKeyboard();
-    adjustWindowSize();
+
+    QRect fullGrabbedRect = screenGrab.rect();
+    QRect nr(QPoint(-1,-1), fullGrabbedRect.size());
```

Comment:
um, since "magic numbers" are used, perhaps there should be added a comment saying that they're fixing a problem?